tests/unit/AnnualReport.spec.ts

```diff
@@ -1251,6 +1255,7 @@ xdescribe('AnnualReport - Part 3B - Submitting (BCOMP)', () => {
     router.push({ name: 'annual-report', params: { filingId: '0' } }) // new filing id
 
     const wrapper = mount(AnnualReport, {
+      sync: false,
```

Comment:
So this little gem helps the unit test flow better and avoid misc issues with loading the DOM or data.
But using this requires some `await flushPromises()` to let things stabilize. `await Vue.nextTick()` doesn't cut it.